SweetTweetApp/src/main/webapp/resourses/js/App.js

```diff
@@ -170,30 +170,186 @@ const POSTS = [
 class App {
 
     constructor() {
-        this.tweetList = new TweetList(POSTS.map((p) => ({ ...p, createdAt: new Date(p.createdAt ) })));
+        //this.tweetList = new TweetList(POSTS.map((p) => ({...p, createdAt: new Date(p.createdAt)})));
+        this.tweetList = new TweetList();
+        this.tweetList.restore();
+        if (this.tweetList._posts.length === 0) {
```

Comment:
It's not a good practice to access "private" property outside of it's class. You can define getter for that